.github/contributing/commits.md

```diff
@@ -16,41 +15,31 @@ to be in a specific format, so they can be parsed.
 
 ### Type
 
-For the type part of the commit, we use the ones defined by the
-[Angular convention](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type)
+The commit type part must be one defined by the [Angular convention](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type)
 
 - **build**: Changes that affect the build system or external dependencies.
 - **ci**: Changes to our CI configuration files and scripts.
-- **chore**: Changes that you don’t want to appear in the change-log. For example this can be used
-  to fix the change-log if any release goes wrong.
+- **chore**: Changes that you don’t want to appear in the change log. For example, this can be used to fix the change log if any release goes wrong.
```

Comment:
```suggestion
- **chore**: Changes that you don’t want to appear in the changelog. For example, this can be used to fix the changelog if any release goes wrong.
```

- [ ] Fix `change log` typos